spec/services/flight_api_service_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'spec_helper'
+require 'json'
+
+def app
+  FlightApiServiceController
+end
+
+describe 'I can connect and recieve a response from the flight API' do
+  it "I get JSON data back" do
+    params = {
+              :fly_from => 'DEN',
+              :date_from => '30/01/2021',
+              :date_to => '30/01/2021',
```

Comment:
FE collects start date and duration from user.  Should we have FE convert the duration to a return date before passing to backend?  Or maybe we do that in backend...

Reply:
We were debating whether it would be easier to use a return date, we also discussed having a way for FE to create a a way for the user to select duration directly on the calendar, similar to how a travel agency does it.

Reply:
We could either do it in the BE, or just make it a return date in the FE instead of duration, that would fix all the problems